### PR TITLE
Fix/nvidia jetson

### PR DESCRIPTION
**Pull request by Luke-1988**

### CODE REVIEW

File: sf-docker/nvidia-jetson/sf_dependencies/docker-compose.yml [hunk 1]
```diff
@@ -3,27 +3,19 @@ services:
...
       - "5432:5432"
     environment:
-      - POSTGRES_PASSWORD=Test1234
-    restart: unless-stopped
+      - POSTGRES_PASSWORD=Test1234    
     volumes:
       - pgsqldata:/var/lib/postgresql/data
       #- /var/lib/smartface/pgsql:/var/lib/postgresql/data
 
-  pgadmin:
```
Comment (Temppus) [inline sf-docker/nvidia-jetson/sf_dependencies/docker-compose.yml:15]: we do not want pgadmin no more ?

File: sf-docker/nvidia-jetson/docker-compose.yml [hunk 2]
```diff
@@ -183,6 +188,9 @@ services:
       - AppSettings__USE_JAEGER_APP_SETTINGS
       - JAEGER_AGENT_HOST
       - AppSettings__Log_RabbitMq_Enabled
+    volumes:
```
[the remark this answers is not in the document]
Comment (Luke-1988) [reply sf-docker/nvidia-jetson/docker-compose.yml:191]: I am not sure, I wanted just to align all configuration. Customers usually points this out during calls

File: sf-docker/nvidia-jetson/sf_dependencies/docker-compose.yml (hunk 1 above)
Comment (Luke-1988) [reply sf-docker/nvidia-jetson/sf_dependencies/docker-compose.yml:15]: Actually I wanted just to save some performance on jetson, however I can rollback if you guys insist on this

**Review by Temppus — APPROVED** [review]

File: sf-docker/nvidia-jetson/docker-compose.yml [hunk 3]
```diff
@@ -169,12 +169,17 @@ services:
 
   sf-graphql-api:
     image: ${REGISTRY}sf-jetson-graphql-api:${SF_VERSION}
+    privileged: true
```
Comment (marek-nereca) [inline sf-docker/nvidia-jetson/docker-compose.yml:172]: why privileged?

File: sf-docker/nvidia-jetson/sf_dependencies/docker-compose.yml (hunk 1 above)
Comment (blaskoa) [reply sf-docker/nvidia-jetson/sf_dependencies/docker-compose.yml:15]: AFAIK pgadmin does not have an arm image, so for jetson the container could not start properly either way... so removing it is OK

File: sf-docker/nvidia-jetson/docker-compose.yml (hunk 3 above)
Comment (Luke-1988) [reply sf-docker/nvidia-jetson/docker-compose.yml:172]: all others are privileged, to be honest I didn't investigate deeply, usually if something is not copy/paste for all in the row, clients point this out and asks

**Review by marek-nereca — APPROVED** [review]

**Pull request merged**